test_particle_in_field.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from turbopy import Simulation
+from particle_in_field import EMWave, ChargedParticle, ForwardEuler
+
+owner_data = {"N": 10, "min": 20, "max": 30, "type": "test_pusher"}
+wave_data = {"amplitude": 10, "omega": 18}
+particle_data = {"position": 26.12, "pusher": "test_pusher"}
+
+
+@pytest.fixture(name="EM_example")
+def fixture_EM():
+    Owner = Simulation({"Grid": owner_data})
+    Owner.read_grid_from_input()
+    return EMWave(Owner, wave_data)
```

Comment:
It looks like you are constructing a minimal "Simulation", and then using it to directly construct an instance of the EMWave class. This is a pretty good approach, since these tests should focus on the PhysicsModules themselves, rather than how they work within a simulation.

However, I'd recommend using one fixture, which returns an instance of the Simulation class. And rather than only calling `read_grid_from_input`, you should probably just call `prepare_simulation`. You can then use this fixture for testing both of the PhysicsModules.